tests/conftest.py

```diff
@@ -95,6 +95,16 @@ def reversal(request):
     return request.param
 
 
+@pytest.fixture(scope="session", params=[True, False], ids=["terminal", "jupyter"])
+def isatty_fixture(request):
+    return request.param
+
+
+@pytest.fixture(autouse=True)
+def isatty_true(monkeypatch):
+    monkeypatch.setattr(sys.stdout, "isatty", lambda: True)
```

Comment:
Unfortunately, this doesn't correctly patch `capsys` so more monkeypatches are needed within test cases that use `capsys`. Do you have another idea?

However, this works fine for a bunch of tests not using `capsys` that would otherwise raise a warning. :+1: